src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignAssembledContigsSpark.java

```diff
@@ -81,7 +73,31 @@ protected void runTool(final JavaSparkContext ctx) {
             }
 
         });
-        assembledBreakpoints.saveAsTextFile(output);
+        allContigAlignments.saveAsTextFile(output);
+    }
+
+    /**
+     * Loads an RDD of {@link ContigsCollection} objects keyed by assembly ID from disk. The input file
+     * should be the output of as RunSGAViaProcessBuilderOnSpark.
+     */
+    static JavaPairRDD<String, ContigsCollection> loadContigsCollectionKeyedByAssemblyId(final JavaSparkContext ctx, final String inputPath) {
+        final JavaRDD<String> inputAssemblies = ctx.textFile(inputPath).cache();
```

Comment:
My understanding of Spark persistence is still pretty limited, so forgive me if this is stupid, but...
a) inputAssemblies is local to this method and referenced only once, so how does caching improve anything?
b) when does the memory get reclaimed and how?  seems like cache() ought to always be matched by unpersist().